x-pack/plugins/security_solution/common/experimental_features.ts

```diff
@@ -260,7 +260,7 @@ export const allowedExperimentalValues = Object.freeze({
   /**
    * Enables the new modal for the value list items
    */
-  valueListItemsModalEnabled: false,
+  valueListItemsModalEnabled: true,
```

Comment:
should test be unskipped in Serverless MKI  and FF removed?

https://github.com/elastic/kibana/blob/main/x-pack/test/security_solution_cypress/cypress/e2e/detection_response/detection_engine/value_lists/value_list_items.cy.ts#L46-L55

Reply:
good point!